rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+This list of labels will be standardized across the organization and will be mandatory, they will apply high level groupings and statuses and will allow for a consistent way of talking about issues across projects 
+
+This consistent vocabulary will enable us to create reporting tools that can be applied to any project using the system and will provide a consistent view into the state of all CDS products. It should also reduce cognitive burden when onboarding to the project. 
+
+The list of proposed baseline labels can be seen in Appendix A
+
+### Product Specific Labels
+
+Each product will be allowed to apply their own unique set of labels alongside the baseline labels. 
+
+
+- ```
+  Name: High Priority | Haute priorité
+  Description: 
+  Colour: 
+- ```
+  Name: Medium Priority  | Priorité moyenne
+  Description:
+  Colour:
+- ```
+  Name: Low Priority | Faible priorité
+  Description:
+  Colour
+- ```
+  Name: Bug | Bogue
+  Description:
+  Colour:
+- ```
+  Name: Documentation
+  Description:
+  Colour:
+- ```
+  Name: Security | Sécurité
+  Description:
+  Colour:
+- ```
+  Name: Privacy | Vie privée
+  Description:
+  Colour:
+- ```
+  Name: Wont fix | Ne résoudra pas
+  Description:
+  Colour:
+
+
+### Label Requirements
+
+Every label is bound by the following requirement:  
+
+- A label must be bilingual in the format `en | fr`
+- A label must not use a colour used by a baseline label
+
+It is recommended that labels make sense for the product/project they are being applied to and that they are not relevant to only a very small number of issues but this isn’t mandatory.
+
+### Governance
+
+Governance surrounding labels will occur in Github repos, as labels are defined as json objects and are enforced through Github actions we can use Github to apply a level of lightweight automated workflows. Github's Code owner functionality will be used to ensure 
+
+#### Creation, Modification, and Approval
+Label Meta-data is defined by Github and can be found here https://docs.github.com/en/rest/reference/issues#create-a-label
+
+The governance surrounding the creation, modification and approval of labels will depend on what label group is being acted upon. 
+
+**CDS wide labels** will be owned by the head of product management. 
+
+**Product Specific Labels** will be owned by the product manager, this is done by applying a .codeowner file against the labels.js file.
+Github Pull requests can be used to provide a lightweight and transparent approval process. 
+
+Any employee of CDS can open a PR or Issue updating the labels, it is recommended that these are documented with a rationale for change.
+
+#### Github Actions
+The head of SRE will own the CDS label repo minus the label.js file which is owned by product managers.
+
+Product specific label repos will be owned by the business unit’s SRE member or the teams 
+
+## Drawbacks
+Why should we _not_ do this?
+
+## Alternatives
+What other designs and processes have been considered?
```

Comment:
The current alternative is what we have now which is some mix of default Github labels and ad-hoc project labels. The downside is getting a holistic view of all the projects and understanding their current status along with the cognitive load of switching projects requiring learning a new set of labels.

Reply:
Good alternative I've added it to the doc